operations-stdlib/src/commonMain/kotlin/string/Replace.kt

```diff
@@ -0,0 +1,86 @@
+package string
+
+import operation.StandardLogicOperation
+import kotlin.runCatching
+import utils.asList
+
+object Replace: StandardLogicOperation, StringUnwrapStrategy {
+    override fun evaluateLogic(expression: Any?, data: Any?): Any? =
+        ReplaceExecutor.buildFromExpression(expression.asList)?.execute()
+}
+
+private data class ReplaceExecutor(
```

Comment:
I like the concept of pure data structures, imo it might be something like simple arguments holder data class and then the execute function(probably the name will change) would turn into some kind extension function inside the replace object. 
I am inspired by the kotlin stdlib and for example map function for Lists

Reply:
I am open to discuss. The opinion from above comes from my intuition

Reply:
I'm not familiar with the style of programming in Kotlin. Sure I will change it.  Inspiring code style from the standard language library is hard to undermine.